### ISSUE TITLE
Hard limit on dvr channels

### ISSUE THREAD

**Issue by M3th0s** [xteve-project/xTeVe#244]
**Describe the bug**
Since the last plex update I’m hitting a hard limit of 77 channels on the dvr. At first I thought I had mapped them wrong and went around hunting for issues. After a while I noticed a pattern and doesn’t matter what tuner it’s coming from or the EPG source there is a max of 77 channels that can be added to the plex DVR.

Could someone please check this and let me know if there is a workaround for it?

Thanks in advance

**Server (please complete the following information):**
- OS: Ubuntu 18.04
- Intel x64
- xTeVe Version: 2.2.0 running on docker

**To Reproduce**
1. Add live TV DVR
2. Select Channels
3. No channels will be added until only 77 are selected, otherwise you get the error "There was a problem saving your DVR. Please try again."

**Expected behavior**
All channels to be added

**Additional context**
I am not certain if this is an xteve or plex issue. However if anyone could help me understand that would be great!

**Comment by winguru** [comment]
I just updated Plex on my Synology to version Version 1.23.1.4602.  I deleted my xTeVe DVR and re-added it and did not encounter this hard limit issue.  What platform are you running Plex on and are you using the Plex Pass version or the public version?  I have a Plex Pass but I only installed the public version update.

Thanks!

**Comment by M3th0s** [comment]
Mine is still a version behind so will update it and let you know :)

**Comment by M3th0s** [comment]
Yeah no dice. Same version has yours and for some reason I can't add all channels... Very frustrating!

**Comment by MCTyler** [comment]
Guy's, This is not Plex. Please refer all Plex questions to Plex.
